src/std/io/socket/api.ss

```diff
@@ -31,23 +32,6 @@
 
```

Comment:
Remove the exports! This was hidden by the GitHub UI.

Reply:
I actually want them, so that 1) you get them automatically from importing `:std/io` and 2) I don't have to hunt down and fix usages -- I have some refactor fatigue.